Kitodo/src/main/java/org/kitodo/production/forms/WorkflowForm.java

```diff
@@ -160,6 +165,26 @@ public String saveAndRedirect() {
         }
     }
 
+    private boolean hasMultipleSructureTreeConfigurations() throws WorkflowException, IOException {
+        Map<String, String> requestParameterMap = FacesContext.getCurrentInstance().getExternalContext()
+                .getRequestParameterMap();
+        String xml = requestParameterMap.get("editForm:workflowTabView:xmlDiagram");
+        xml = StringUtils.substringBefore(xml, "kitodo-diagram-separator");
+        Converter converter = new Converter(new ByteArrayInputStream(xml.getBytes(StandardCharsets.UTF_8)));
+        List<Task> tasks = converter.validateWorkflowTaskList();
+        boolean separate = false;
+        boolean combined = false;
+        for (Task task : tasks) {
+            if (task.isSeparateStructure()) {
+                separate = true;
+            } else {
+                combined = true;
+            }
+        }
+
+        return separate && combined;
```

Comment:
This could simplified to something like this:
```        
Set<Boolean> configurations = converter.validateWorkflowTaskList()
                .stream().map(Task::isSeparateStructure).collect(Collectors.toSet());
return configurations.size() > 1;
```